FileSystem/Workbook.js

```diff
@@ -0,0 +1,23 @@
+
+var FileSystem = {
+  current_directory: "<ToBeDefined>",
```

Comment:
Add a todo here to address current_directory work.

Reply:
Done